### ISSUE TITLE
Dirt with Primordial Mycelium is not photosensitive

### ISSUE THREAD

**Issue by alerikaisattera** [FaceDeer/dfcaverns#28]
Dirt with Primordial Mycelium is not photosensitive but seems like it should be

**Comment by FaceDeer** [comment]
I think this bug is a leftover from an old idea I toyed with but ultimately didn't implement; the Primordial ecological apocalypse. :) I figured, there must be a reason why the primordial layers are sealed away and so dangerous to access. Some kind of demon mobs living down there might be the obvious approach, but I wanted something a little more subtle and creative than that. So I was going to have it so that if a player transplanted the giant mycelia or the primordial plant matter from the primordial layer to the surface world it would start spreading and taking over the landscape as an invasive organism. I was having a hard time coming up with a way to make that happen without it being obnoxious, though. I ultimately decided I was trying to balance two mutually contradictory outcomes and cancelled the apocalypse.

Anyway, I'll add sunlight sensitivity. It'll have to be low sensitivity, though, the primordial fungal caverns are surprisingly bright.
